pallets/exchange/benchmarking/src/mock.rs

```diff
@@ -29,11 +29,11 @@ use sp_runtime::{
 };
 
 use frame_system::EnsureSigned;
+use hydradx_traits::AssetPairAccountIdFor;
```

Comment:
probably not for this PR but @enthusiastmartin do we want to move this to gc/warehouse?

Reply:
hydradx traits are already from warehouse.

Reply:
Yes. I always wondered what it even means:))) but It has been there since the very beginning. 

I will create issue in warehouse